### PR TITLE
CI: Add Coverity notification email.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add an encrypted COVERITY_SCAN_NOTIFICATION_EMAIL address for Coverity
Scan notifications.

Signed-off-by: James Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ env:
...
+    # The next declaration is the encrypted COVERITY_SCAN_NOTIFICATION_EMAIL, created
+    #   via the "travis encrypt" command using the project repo's public key
+    - secure: qYWd0Njhqt3w4b3Dvj38pWzGEdHQaaodJDOy8QMxb+g3v3VIbn5WajEGd1t4J8S7+hPl+Hgo1bs96PoUDXAm29FAL7VAaqGkG2+RuWmGN3PZbdOmnDmamzuz4IN04VG6O3WQpdp6aZdS+BL8VnNllbxmyKS3rFOAB6I9YRj7XQAbLgwfWEnRwy+e4d9mvIxJ/aEMkhGX4qbquFvnqBXUu/YRq8DXi0Pqi/DPqGHmx+YNI0bmgRLGkoC/9W3YL9MMnc14lK9sCiJfio9onsm7uMtQlDbR5uAd25oM/TV1Pay53T63JTsZMbRKW6+7mWr7BLeWkUX98PmxlQD4cumcc3J8jIXp7lUZ9fRUDeQ6i+4uR4rf/mtVYg2AKKDNpvLqIgEH8T3dDHJdlBg8fzd3v1MhC21Y4Brri6fWtphkaHocNuWUUj8XZDr/f4LPSYRfulgT+Y/Xz79YzDN4jxXcrLhS9XVX4CEzQ5rHsCYzJHnTDK0+vQsJdcHHgxOmhgL1ScGEeSjjWucAEeWyvgiDTpYvUDwO91OTcpjhgzQXzCO/f9dIe/MwUmJThYUOd0jGbrpUdgS46mhzEnKercuMky+cHIMIIfkCQBR1oWxpxFUEGVQ4NHAjrqtW5SpzlUsO2xmn3rSDiM3UAmuWvOGCREq6YGQ0lzzS195iRgWjme4=
```
Comment (dlespiau) [inline .travis.yml]: This only doesn't have the quotes to delimit a string that the other has. A problem in either of those?

**Comment by dlespiau** [comment]
lgtm with the question addressed. Maybe both are fine.

**Comment by jodh-intel** [comment]
Good catch - quotes added.

**Comment by dlespiau** [comment]
git commit -a --amend and git push -f :)

**Comment by jodh-intel** [comment]
Collapsed into a single commit now.

**Comment by dlespiau** [comment]
Thanks, lgtm holds, obviously.

**Comment by grahamwhaley** [comment]
lgtm - will merge.

**Pull request merged**